python/ray/rllib/models/visionnet.py

```diff
@@ -12,11 +12,13 @@ class VisionNetwork(Model):
     """Generic vision network."""
 
     def _init(self, inputs, num_outputs, options):
+        # we must adjust for downscaling done in preprocessors
+        f = options.get("downscale_factor", 2)
         with tf.name_scope("vision_net"):
-            conv1 = slim.conv2d(inputs, 16, [8, 8], 4, scope="conv1")
+            conv1 = slim.conv2d(inputs, 16, [16//f, 16//f], 4, scope="conv1")
```

Comment:
I think it is preferable to have {"conv_filtersizes": [(16, 16), (4, 4), (20, 20)]} or similar as part of the "model" config entry here, because in general we will want to experiment with different architectures here.